### DISCUSSION TITLE
If an origin sends a `Connection: close` does the individual connection leave the Pool?

### DISCUSSION THREAD

**Discussion by Ethan-Arrowood** [nodejs/undici#1231] [Q&A] [answered]
Given an `Agent`, when an origin returns a `Connection: close` header, does the entire connection `Pool` terminate, or only the individual connection within the `Pool` for that origin?

**Comment by ronag** [comment, marked as answer]
Only the individual connection.

**Reply by Ethan-Arrowood** [reply]
Thank you!
